### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by lyskoivan**

### CODE REVIEW

File: src/routes/router.js [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+    return productsByCategory;
+  }
+  if (url === "/products") {
+    return productsRoute;
+  }
+  if (url === `/products/${productId}`) {
+    return idProductRoute;
+  }
+  if (url === "/signup") {
+    return signupRoute;
```
Comment (lyskoivan) [inline src/routes/router.js]: is it better to rewrite in to the object?

File: src/routes/router.js [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
+  }
+  return mainRoute;
 };
```
Comment (lyskoivan) [inline src/routes/router.js]: is it better to rewrite in to the object?

File: src/routes/products/allIds.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  );
+
+  fs.readFile(productsPath, "utf8", (err, data) => {
+    try {
+      const allProducts = JSON.parse(data);
+      const reqUrl = URL.parse(req.url);
+      const queryStr = querystring.parse(reqUrl.query);
+      const queryIds = queryStr.ids
+        .slice(1, queryStr.ids.length - 1)
+        .split(",");
```
Comment (lyskoivan) [inline src/routes/products/allIds.js:23]: didn't find a way parse URL with quotes without wrapper into the one more quotes

File: src/routes/router.js (hunk 2 above)
Comment (odykyi) [reply src/routes/router.js]: yes, rewrite to object

**Review by odykyi — APPROVED** [review]

**Pull request merged**